shell/browser/ui/electron_desktop_window_tree_host_linux.cc

```diff
@@ -241,4 +242,24 @@ void ElectronDesktopWindowTreeHostLinux::UpdateFrameHints() {
     SizeConstraintsChanged();
   }
 }
+
+void ElectronDesktopWindowTreeHostLinux::DispatchEvent(ui::Event* event) {
+  if (event->IsMouseEvent()) {
+    auto* mouse_event = static_cast<ui::MouseEvent*>(event);
+    bool should_disable_drag =
+        mouse_event->IsRightMouseButton() ||
+        (mouse_event->IsLeftMouseButton() && mouse_event->IsControlDown());
```

Comment:
Is ctrl + leftmouse something linux apps support similar to macOS ? haven't tested, just wanted to double check.

Reply:
@deepak1556 my understanding is yes - https://forums.linuxmint.com/viewtopic.php?t=404292

> you can configure your Linux Mint system to make the Control key on the keyboard act as a right-click when combined with a left-click from the mouse